src/spikeinterface/core/recording_tools.py

```diff
@@ -662,9 +761,11 @@ def get_noise_levels(
         The method to use to estimate noise levels
     force_recompute : bool
         If True, noise levels are recomputed even if they are already stored in the recording extractor
-    random_chunk_kwargs : dict
-        Kwargs for get_random_data_chunks
-
+    random_slices_kwargs : dict
+        Options transmited to  get_random_recording_slices(), please read documentation from this
+        function for more details.
+    **job_kwargs: 
```

Comment:
I would actually copy/paste the kwargs in the docstring here, since it's much higher level API